### PR TITLE
CHANGE (GraphEngine): @W-11999008@: Refactoring appropriate methods into PathEntryPointUtil.

### PR DESCRIPTION
**Pull request by jfeingold35**

### CODE REVIEW

File: sfge/src/test/java/com/salesforce/graph/ops/MethodUtilTest.java [hunk 1]
```diff
@@ -334,136 +329,4 @@ public void getTargetMethods_targetMethodInInnerAndOuterClass() {
...
-
-    @ValueSource(
-            strings = {
-                Schema.AURA_ENABLED,
-                Schema.INVOCABLE_METHOD,
-                Schema.REMOTE_ACTION,
-                Schema.NAMESPACE_ACCESSIBLE
-            })
-    @ParameterizedTest(name = "{displayName}: {0}")
-    public void testGetMethodsWithAnnotation(String annotation) {
```
Comment (rmohan20) [inline sfge/src/test/java/com/salesforce/graph/ops/MethodUtilTest.java:346]: Could you point me to where these tests now live?

Comment (jfeingold35) [reply sfge/src/test/java/com/salesforce/graph/ops/MethodUtilTest.java:346]: Line 325 of `PathEntryPointUtilTest.java`.

File: sfge/src/main/java/com/salesforce/graph/ops/PathEntryPointUtil.java [hunk 2]
```diff
@@ -64,5 +78,246 @@ public static boolean isPathEntryPoint(MethodVertex methodVertex) {
...
+                .repeat(__.out(Schema.CHILD))
+                .until(__.hasLabel(ASTConstants.NodeType.METHOD))
+                .not(has(Schema.IS_TEST, true));
+    }
+
+    /**
+     * Returns non-test methods in the target files whose modifier scope is `global`. An empty list
+     * implicitly includes all files.
+     */
+    static List<MethodVertex> getGlobalMethods(GraphTraversalSource g, List<String> targetFiles) {
```
Comment (rmohan20) [inline sfge/src/main/java/com/salesforce/graph/ops/PathEntryPointUtil.java:225]: private?

File: sfge/src/main/java/com/salesforce/graph/ops/PathEntryPointUtil.java [hunk 3]
```diff
@@ -64,5 +78,246 @@ public static boolean isPathEntryPoint(MethodVertex methodVertex) {
...
+                                        // We only want global methods.
+                                        out(Schema.CHILD)
+                                                .hasLabel(ASTConstants.NodeType.MODIFIER_NODE)
+                                                .has(Schema.GLOBAL, true),
+                                        // Ignore any standard methods, otherwise will get a ton of
+                                        // extra results.
+                                        __.not(__.has(Schema.IS_STANDARD, true)))));
+    }
+
+    static List<MethodVertex> getInboundEmailHandlerMethods(
```
Comment (rmohan20) [inline sfge/src/main/java/com/salesforce/graph/ops/PathEntryPointUtil.java:248]: private?

File: sfge/src/main/java/com/salesforce/graph/ops/PathEntryPointUtil.java [hunk 4]
```diff
@@ -64,5 +78,246 @@ public static boolean isPathEntryPoint(MethodVertex methodVertex) {
...
+
+    /**
+     * Returns all non-test public- and global-scoped methods in controllers referenced by
+     * VisualForce pages, filtered by target file list. An empty list implicitly includes all files.
+     *
+     * @param g
+     * @param targetFiles
+     * @return
+     */
+    static List<MethodVertex> getExposedControllerMethods(
```
Comment (rmohan20) [inline sfge/src/main/java/com/salesforce/graph/ops/PathEntryPointUtil.java:280]: private?

**Pull request merged**